### PR TITLE
Tag AzureClient public

### PR DESCRIPTION
**Pull request by anthony-murphy**
Tags the AzureClient class and all it's transitive dependencies as public.

### CODE REVIEW

File: packages/dds/map/api-report/map.api.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ export interface ICreateInfo {
     csn: number;
 }
 
-// @alpha
+// @internal
```
Comment (ChumpChief) [inline packages/dds/map/api-report/map.api.md:47]: Is it expected for items to become more private in this PR?  I would have expected this one to remain `@alpha` at least?

File: packages/common/container-definitions/api-report/container-definitions.api.md [hunk 2]
```diff
@@ -36,22 +36,22 @@ import { IUsageError } from '@fluidframework/core-interfaces';
...
+// @public
 export enum AttachState {
     Attached = "Attached",
     Attaching = "Attaching",
     Detached = "Detached"
 }
 
-// @alpha
+// @public
 export namespace ConnectionState {
```
[the remark this answers is not in the document]
Comment (anthony-murphy) [reply packages/common/container-definitions/api-report/container-definitions.api.md:47]: https://dev.azure.com/fluidframework/internal/_workitems/edit/6608

File: packages/framework/fluid-static/src/fluidContainer.ts [hunk 3]
```diff
@@ -31,7 +31,8 @@ export type InitialObjects<T extends ContainerSchema> = {
 
 /**
  * Events emitted from {@link IFluidContainer}.
```
Comment (Josmithr) [inline packages/framework/fluid-static/src/fluidContainer.ts:33]:
```suggestion
 * Events emitted from {@link IFluidContainer}.
 *
 * @remarks Note: external implementations of this interface are not supported.
```

Comment (Josmithr) [reply packages/framework/fluid-static/src/fluidContainer.ts:33]: (since use of `@sealed` on interfaces is somewhat nonstandard.

**Review by Josmithr — APPROVED** [review]

**Pull request merged**